site/viamguides/src/workshop-plant-watering/workshop-plant-watering.md

```diff
@@ -0,0 +1,506 @@
+  - **Provide network information for the machine**: Go to the provisioning portal at `http://viam.setup/` in a browser to follow [the on-screen instructions](https://docs.viam.com/fleet/provision/#end-user-experience) to configure the Raspberry Pi for your home Wi-Fi.
+
+- **Remote control**: If participants cannot take their device home but want to continue their projects, they can access it remotely:
+
+  - **Access the workshop device**: Participants can control the workshop device remotely from anywhere in the world. A webcam can be set up for participants to view the device activation in real time.
+  - **Code from home**: Without SSH access to the Raspberry Pi, participants can still write and test code on their laptops and integrate changes during their next in-person session or via workshop-provided tools.
+
+<!-- ------------------------ -->
+
+## Post-Workshop Resources
+
+Duration: 5
+
+### Group discussion topics 🗣️
+
+Review suggested topics for discussion at key points during the workshop.
+
+- What were the biggest challenges participants faced during the workshop (e.g., assembly, coding, debugging)?
+- Share troubleshooting experiences and how participants resolved issues.
+- What additional features could improve the system (e.g. sensors, cloud services, mobile app)?
+- How can automated plant watering be used in homes, schools, or offices?
+- Discuss different ways a plant watering device can interact with the environment in industrial settings and outdoor spaces.
+- Discuss the feasibility of deploying a network of automated plant watering devices for a citywide project.
+- How can the design of the device be improved for better accessibility, performance, or durability?
+
+### Quiz questions ❓
+
+Review the suggested quiz questions below to evaluate participants' understanding of key concepts. These questions can be used to assess knowledge either during (see multiple choice answers in the delivery plan) or after the workshop.
+
+1. **Hardware Assembly**
+   - What is the purpose of setting a custom `hostname` during the Raspberry Pi OS configuration?
+   - What hardware signals are used to indicate with the soil moisture is too low?
+1. **Software Setup**
+   - Why is it important to connect your machine to the Viam app during setup?
+   - What does it mean that `viam-server` enables remote access to the Raspberry Pi?
+   - How does the Viam app simplify the process of testing and controlling hardware components like the board?
+1. **Hands-on Experiment**
+   - What is the primary role of `viam-server` in the system architecture of your plant watering setup?
+   - What is the purpose of the `env` field in your process configuration?
+
+### Next-level projects
+
+Here are some ideas for expanding and enhancing the current project to create autonomous behaviors.
```

Comment:
Consider adding a link to this project (Nicolai's Viam video): https://youtu.be/l4MQRiHPY6Y?feature=shared